modules/virt-importing-vm-datavolume.adoc

```diff
@@ -77,8 +77,8 @@ spec:
         storageClassName: local
       source:
         http: <1>
-          url: "https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2" <2>
-          secretRef: "" <3>
+          url: <https://mirror.arizona.edu/fedora/linux/releases/35/Cloud/x86_64/images/Fedora-Cloud-Base-35-1.2.x86_64.qcow2> <2>
```

Comment:
IIUC, this should still use quotation marks instead of `< >` because it's an actual example URL, not something like `<image_url>`. This also applies to the next file.